### DISCUSSION TITLE
Tell Renovate to try and propose a lower version next time

### DISCUSSION THREAD

**Discussion by chopeen** [renovatebot/renovate#20062] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

My application uses `sagemaker v2.117.0`. Renovate proposes to update it to `sagemaker v2.130.0` (latest version of the package), but other packages in my application are not compatible with `v2.130.0`.

Is there a way to tell Renovate to try a lower version (i.e. `v2.129.0`) next time it runs?
Alternatively, is there a way to tell it to create MRs for all versions between the current and the latest (i.e. `v2.117.0 - v2.130.0`)?

**Comment by rarkins** [comment]
I'm not sure if we have a feature request for the first idea. It's been discussed before, but I don't think we found a viable solution. For example there could be 1000s of releases between current and latest in some cases, and we can't list them all in a PR body to let the user pick.

For the second question, see #6576
